beta/project-griffon/using-project-griffon/validation-summary-and-project-griffon.md

```diff
@@ -57,7 +57,7 @@ To execute the function in the code editor, click **Test**. The function is expe
 | :--- | :--- | :--- | :--- | :--- |
 | `message` | String | The validation message to display in the summary results. |  |  |
 | `events` | Array | An array of event uuids to be reported as matched or not matched. |  |  |
```

Comment:
adjust table columns (removing the last 2 columns).

Reply:
fixed